orttraining/orttraining/python/training/ortmodule/runtime_options.py

```diff
@@ -0,0 +1,216 @@
+        return self._use_static_shapes
+
+    @property
+    def exporter_extra_args(self):
+        """Accessor for extra arguments to the exporter."""
+
+        return self._exporter_extra_args
+
+
+class RuntimeOptions:
+    """Configurable runtime options for ORTModule.
+
+    Args:
+        onnx_opset_version (:obj:`int`, optional, default 14): Configure ORTModule exporter to use this ONNX opset version..
+            To change the opset version, the environment variable "ORTMODULE_ONNX_OPSET_VERSION" can also be used.
+
+        do_constant_folding (:obj:`bool`, optional): Configure ORTModule export to apply the constant-folding optimization.
+            Defaults to False. Constant-folding will replace some of the ops that have all constant inputs with pre-computed
+            constant nodes.
+
+        export_modules_as_functions (:obj:`bool` or set of python:type of nn.Module, optional, default False): ORTModule exporter
+            flag to enable exporting all nn.Module forward calls as local functions in ONNX. Or a set to indicate the particular
+            types of modules to export as local functions in ONNX. This feature requires onnx_opset_version >= 15, otherwise the
+            export will fail. This is because onnx_opset_version < 15 implies IR version < 8, which means no local function support.
+
+        export_params (:obj:`bool`, optional, default False): ORTModule exporter flag to export all model parameters.
+            If True, all parameters will be exported. Set this to False if you want to export an untrained model. In this case,
+            the exported model will first take all of its parameters as arguments.
+
+        keep_initializers_as_inputs (:obj:`bool`, optional, default True): ORTModule exporter flag to keep initializers as inputs.
+            If True, all the initializers (typically corresponding to parameters) in the exported graph will also be added as inputs
+            to the graph. If False, initializers are not added as inputs to the graph, and only the non-parameter inputs are added as inputs.
+            This may allow for better optimizations (e.g. constant folding) by backends/runtimes.
+
+        use_static_shapes (:obj:`bool`, optional, default False): ORTModule exporter flag to use static output shapes.
+            By default the exported model will have dynamic shapes for all input and output tensors.
+
+        exporter_extra_args (:obj:`dict`, optional, default Empty): Extra arguments to specify to the ORTModule Torch ONNX exporter.
+            Refer to https://pytorch.org/docs/stable/onnx.html#torch.onnx.export for documentation on exporter arguments.
+
+        enable_custom_autograd (:obj:`bool`, optional, default False): Enable custom autograd.Function support for ORTModule.
+            This will enable ``autograd.Function``s to be exported as ``PythonOp`` in ONNX.
+
+        disable_custom_ops (:obj:`bool`, optional, default False): Disable custom ops support in ORTModule.
```

Comment:
what is the intention of this option? Do you want to disable aten op overrides or aten ops altogether? 

Some Context: Today the only way to enable or disable aten ops is via the symbolic registry... soon we will enable the ATEN_FALLBACK option in exporter (which will be available starting pytorch 1.12)... with this change, when this option is selected any op which cannot be represented with a single onnx op or a combination of onnx ops will be exported as ATEN op. 

Going back to my question - do you want to disable just the overrides or turn off aten op export?

Reply:
The intention behind introducing this option is to be able to turn off ATen op export in the graph as external/custom EPs may not fully support them. Is there a way to disable them in the exporter?